bigquery_etl/backfill/shredder_mitigation.py

```diff
@@ -23,11 +23,12 @@
 from bigquery_etl.util.common import extract_last_group_by_from_query, write_sql
 
 PREVIOUS_DATE = (dt.now() - timedelta(days=2)).date()
-SUFFIX = dt.now().strftime("%Y%m%d%H%M%S")
 TEMP_DATASET = "tmp"
 THIS_PATH = Path(os.path.dirname(__file__))
 DEFAULT_PROJECT_ID = "moz-fx-data-shared-prod"
 QUERY_WITH_MITIGATION_NAME = "query_with_shredder_mitigation"
+WILDCARD_STRING = "???"
+WILDCARD_NUMBER = -999
```

Comment:
Are these values only for the intermediate steps and don't show up in the final output?  If so, it might be worth making these more random/larger to reduce the chance of a collision with actual values

Reply:
Great idea, I've made them larger.